.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=ROHAN
```

Comment:
Add your .env back to gitignore, as you should never publicly push your keys to public